learning_resources_search/tasks.py

```diff
@@ -660,6 +689,81 @@ def start_recreate_index(self, indexes, remove_existing_reindexing_tags):
     )
 
 
+@app.task(bind=True)
+def start_embed_resources(self, indexes, skip_content_files):
```

Comment:
Add a unit test?

Reply:
added test for the task and for exiting out of embedding without the settings specified